scripts/sync-project-board.sh

```diff
@@ -0,0 +1,124 @@
+#!/usr/bin/env bash
+
+# Copyright 2022 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Vendored from kubernetes-sigs/sig-windows-tools/hack/sync-project-board.sh
+#  * commit: 09de175041779f35595dee339c2463b2eb194555
+#  * link: https://github.com/kubernetes-sigs/sig-windows-tools/blob/09de175041779f35595dee339c2463b2eb194555/hack/sync-project-board.sh
+
+
+set -e
+set -u
+set -o pipefail
+
+## DESCRIPTION:
+##
+## This script queries all repos in a given github org and adds the issues 
+## with label 'sig/auth' to a specified project board.
```

Comment:
Meta question: should this be a simple `go` program that we just `go run`?  I feel like every time we write a bash script in Kube we quickly outgrow it...

Note that I do not mean anything fancy: just a single `main.go` file with a `main` func (and `go.mod`).